### ISSUE TITLE
Content strategy & content for STEM page

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#11804]
## Issue Description
We need to create a page to bring over STEM scholarship content to VA.gov and support the new STEM scholarship form.

---
## Tasks
- [x] SEO keyword research (Danielle)
- [x] Rough draft content (Danielle)
- [x] Finalize content strategy/create modernized content for page (Danielle)
- [ ] Edit content (Beth)

## Acceptance Criteria
- [ ] Content ready to be built in Drupal

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by DanielleThierryUSDSVA** [comment]
Hi @mnorthuis. Here's a rough working draft of this page for us to do a content strategy/IA sync on. I've included SEO keyword research at the top. Also uploading research results spreadsheet. Could we sync this week even for 15 minutes to make sure we're aligned? Thanks!

[STEM scholarship page_working draft_8.4.20](https://app.zenhub.com/files/133843125/d08e7c1c-62cf-479f-99ed-35ac15dfaebf/download)

[va_stem_scholarship-phrase_related-us](https://app.zenhub.com/files/133843125/47bc9bab-872f-4cfc-a3fe-86beb70d7dd0/download)

**Comment by mnorthuis** [comment]
Thank you @DanielleSoCompany! 

Some additional content I think needs to be done that I want to discuss:
Education hub page - update teaser text on Other Education benefits option to reflect the addition of a scholarship
Other education benefits page - re-order the options and ensure a proper call-out to all other education benefits outside of GI Bill core benefits

Beyond that, we should align on entry points and crosslinking.

**Comment by DanielleThierryUSDSVA** [comment]
## Notes from meeting (Danielle & Mikki)

- Revised Education hub page teaser to be more general and include mention of stem
- Update Other education benefits page to be a straightforward link list not associated with "if you don't qualify for post-9/11; move GI Bill comparison tool & accredited rep links to related links section; revisit REAP note
- Add crosslinks from post-9/11, fry scholarship, and undergrad degree pages

**Comment by DanielleThierryUSDSVA** [comment]
@bethpottsVADEPO Here's the page to edit. When it's ready, could you please post in https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/11807 and ask Randi to assign to a content team member to build in Drupal by 8.14? Thank you!

[STEM scholarship page_working draft_8.4.20](https://app.zenhub.com/files/133843125/c9d604f1-40c2-40e1-b1d2-8870c7e75ab4/download)

**Comment by DanielleThierryUSDSVA** [comment]
**Note for sprint planning:** Once Beth is finished editing, this can be closed. I'm opening another ticket for the related page updates. Thanks!

**Comment by bethpottsVADEPO** [comment]
@RLHecht @MickinSahni Here's the text for building this page in drupal. I'm uploading a version with my  tracked changes so Danielle can see what I did when she returns. For the sake of building the page, go ahead and accept all edits.[https://app.zenhub.com/files/133843125/e654d29a-f119-45d5-a406-67d654d3a564/download](https://app.zenhub.com/files/133843125/e654d29a-f119-45d5-a406-67d654d3a564/download)

**Comment by bethpottsVADEPO** [comment]
Closing as we've moved on to the drupal stage of the process.
